content/organizing-faq.md

```diff
@@ -0,0 +1,31 @@
+
+# Can I call it a Cloud Native Day, Mini-KubeCon, or similar?
+
+Sorry, but it needs to be called Kubernetes Community Day [Location] (e.g., Kubernetes Community Day Bouvet Island). There are two reasons. First, we need it to include the term Kubernetes, as CNCF's ownership of the trademark lets us maintain control. Specifically, cloud native is not trademarked so anyone can run a Cloud Native Day, but Kubernetes Community Days can only operate under CNCF's aegis. Second, we believe that there will be a lot of value over time in building up the Kubernetes Community Day brand, in that it will enable new cities to more easily recruit speakers, sponsors, and attendees.
+
+# Why is CNCF mandating the use of specific tools like Eventbrite for registration, Sched for the schedule, and SM Apply for the CFP?
+
+At launch, we want to ensure that KCDs use a set of software that CNCF staff is familiar with and can help support, and that enables KCD organizers from different cities to support each other. Additionally, we understand the privacy policies of the tools we're using, and we're able to ensure that each organizer in a city is on the same footing.
+
+We acknowledge that this makes being a KCD organizer more difficult for employees of companies where they've already standardized on other tools. However, we think this is a worthwhile tradeoff in ensuring that all organizers have the same access and familiarity and enabling both community and CNCF support. Also, note that CNCF is paying for commercial licenses for each of these tools that cover all KCD usage.
+
+The exception will be in places like China where the standard tools are blocked or don't support local payment options. There, we'll of course need to work on alternatives. CNCF is open to expanding the number of supported systems after KCDs are up and running (e.g., in 2021). 
+
+# Is CNCF underwriting these events or does that fall on the organizers?
+
+The organizers are taking on the risk for running the events.
+
+# Will CNCF host the website or does one of the organizers have to take ownership?
+
+Organizers are responsible for creating the content, but CNCF hosts the website. Lots of details are in the [creating website](https://kubernetescommunitydays.org/organizing-creating-website/) page. In particular, we don't allow KCDs to have a separate website, because we want all changes to come through the KCD GitHub repo so that we can review changes and engage if there is any problematic content.
+
+# What corporate entity should sponsors sign a contract with?
+
+DevOpsDays has been successful bringing on sponsors without having them sign [contracts](https://devopsdays.org/sponsor/). Although we don't forbid organizers from signing a contract (such as by having their employer as the counterparty), we recommend against it.
```

Comment:
Is this a change?  In December CNCF folks were sharing a sample sponsor contract in the kcd-organizers channel.

Also, some sponsors *require* paperwork, particularly in the US.  So just saying "we recommend against it" isn't adequate.

Reply:
There has been some confusion between the prospectus versus a contract. We don't encourage folks to get into the contract signing business. For DevOpsDays, the process is by far no contracts at all. Some sponsors try to do contracts, but when faced with "no we don't do that" concede.